### ISSUE TITLE
RTCDataChannel.send(null)

### ISSUE THREAD

**Issue by lgrahl** [w3c/webrtc-pc#1835]
I'm unsure what's supposed to happen when calling `dataChannel.send(null)`. Step 3 of the `send` method states:

> Execute the sub step that corresponds to the type of the methods argument:
> ...

But what if none of the types match?

**Comment by alvestrand** [comment]
TypeError seems right to me for arguments that have no substep defined.
For the particular case of null, returning without action might be equally appropriate.

**Comment by lgrahl** [comment]
Then, I would come back and ask the same question for `undefined`. :)
But `TypeError` sounds good to me. Should we explicitly mention that in the spec as a *catch all* step?

**Comment by adam-be** [comment]
I think this is already covered by WebIDL since none of the send method overloads have a nullable argument. The result will be TypeError.

**Comment by lgrahl** [comment]
I agree but since all implementations currently get it wrong, I think it's worth spending an extra step to provide clarity.

**Comment by adam-be** [comment]
Would a note be sufficient?

**Comment by aboba** [comment]
@lgrahl  Can you submit a PR?
